ops/packaging/Web-Dockerfile

```diff
@@ -14,6 +14,8 @@ COPY images /var/www/images
 COPY index.php /var/www/index.php
 COPY js /var/www/js
 COPY less /var/www/less
+COPY files /var/www/files
+COPY uploads /var/www/images/uploads
```

Comment:
@lump1k `COPY uploads /var/www/images/uploads` is breaking my local Docker build because the `gigadb-website/uploads` directory is not present.

Any changes to Dockerfiles should be briefly summarised in the `Changes to the provisioning` section in the pull request.

Reply:
COPY uploads /var/www/images/uploads
I removed it